### PR TITLE
Update lucuma-schema to v0.4.0 - changes to the `execution` queries

### PR DESCRIPTION
**Pull request by toddburnside**
The sequence decoders are now in `lucuma-odb` and are tested there.

### CODE REVIEW

File: lucuma-schemas/src/clue/scala/lucuma/schemas/odb/SequenceQuery.scala [hunk 1]
```diff
@@ -51,20 +48,12 @@ object SequenceSQL:
                   science {
                     ...gmosSouthSequenceFields
                   }
-                  setup {
```
Comment (rpiaggio) [inline lucuma-schemas/src/clue/scala/lucuma/schemas/odb/SequenceQuery.scala:54]: Setup was moved to digest?

Comment (rpiaggio) [reply lucuma-schemas/src/clue/scala/lucuma/schemas/odb/SequenceQuery.scala:54]: I just had to continue scrolling :D I see it in the digest.

**Review by rpiaggio — APPROVED** [review]
LGTM

File: lucuma-schemas/src/clue/scala/lucuma/schemas/odb/SequenceQuery.scala (hunk 1 above)
Comment (toddburnside) [reply lucuma-schemas/src/clue/scala/lucuma/schemas/odb/SequenceQuery.scala:54]: Yes

**Pull request merged**